### PR TITLE
Change method name in DNSAAS api

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/dns/dnsaas.py [hunk 1]
```diff
@@ -153,9 +153,9 @@ def create_dns_records(self, record):
         if request.status_code != 201:
             return request.json()
 
-    def delete_dns_records(self, record_id):
+    def delete_dns_record(self, record_id):
         """
-        Delete rcords in DNSAAS
+        Delete rcord in DNSAAS
```
Comment (ghost) [inline src/ralph/dns/dnsaas.py]: `rcord` -> `record`?

Comment (ar4s) [inline src/ralph/dns/dnsaas.py]: rcord or record?

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**